include/lldb/Symbol/SwiftASTContext.h

```diff
@@ -867,6 +867,10 @@ class SwiftASTContext : public TypeSystem {
   bool m_initialized_search_path_options = false;
   bool m_initialized_clang_importer_options = false;
   bool m_reported_fatal_error = false;
+
+  // Whether this is a scratch or a module AST context.
+  bool m_is_scratch_context = false;
```

Comment:
Does the scratch context have llvm isa support? Because it is always a SwiftASTContextForExpression, a subclass of SwiftASTContext.